ui/k-info-panel/list_connections.kytos

```diff
@@ -0,0 +1,97 @@
+<template>
+    <k-table title="list of EVCs"
+        :headers="headers"
+        :rows="rows">
+    </k-table>
+
+</template>
+<script>
+module.exports = {
+  data(){
+    return {
+        headers: ['Name ', 'Endpoint A' , 'Tag A' , 'Endpoint Z' , 'Tag z' , 'Enabled' , 'Active'],
+
+        rows: this.list_EVCs()
+    }
+  },
+   mounted () {
+     console.log('inside the list connections: ' )
+   },
+  methods: {
+
+    list_EVCs()
+  {
+    var tableDataArr = [];
+
+     $.ajax(
+     {
+        url: this.$kytos_server_api + "kytos/mef_eline/v2/evc/",
+        type:"GET",
+        data: JSON.stringify(),
+        dataType: "json",
+        contentType: "application/json; charset=utf-8",
+        success: function(data, textStatus){
+            $.each(data, function(i , evc){
+
+                var connection = {
+                    "Name" : evc.name,
+                    "Endpoint A": evc.uni_a.interface_id,
+                    "Tag A": evc.uni_a.tag.value,
+                    "Endpoint Z": evc.uni_z.interface_id,
+                    "Tag Z": evc.uni_z.tag.value,
+                    "Enabled" : evc.enabled,
+                    "Active" : evc.active
+                };
+                tableDataArr.push(connection);
+            });
+
+            console.log(tableDataArr)
+            console.table(tableDataArr)
+
+        },
+        error: function(data){
+ //           console.log("GET request Error ("+data.status+"):"+ data.responseJSON.description;
+       },
+     })
+
+     return tableDataArr;
+    }
+
+  }
+}
+</script>
+<style>
```

Comment:
The k-table will have its own CSS updated in our ui, so there is no need for this to be included here, I think that should be removed of this PR.